src/Microsoft.Azure.WebJobs.Logging/Internal/Utility.cs

```diff
@@ -72,5 +74,80 @@ public static string DecodeBase64(string base64)
             string str = Encoding.UTF8.GetString(bytes);
             return str;
         }
+
+
+        private static async Task SafeCreateAsync(this CloudTable table)
```

Comment:
I'll change it to have the logging stop eagerly creating the tables.  That will then mean all the tests will implicitly test the auto-create path.